crafters/nlp/TikaExtractor/__init__.py

```diff
@@ -0,0 +1,60 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import requests
+from requests.exceptions import ConnectionError
+from jina.executors.crafters import BaseCrafter
+
+TIKA_URL = 'http://0.0.0.0:9998'
+
+
+class TikaExtractor(BaseCrafter):
+    """
+    :class:`TikaExtractor` Extracts text from files.
+    """
+
+    def start_tika(self):
```

Comment:
How will this process be finished? Should there be some handling in the overriden close method?

Reply:
Killed the process now in the overriden `close` method.